cumulus_library_data_metrics/q_date_in_lifetime/patient_lifespan.jinja

```diff
@@ -0,0 +1,8 @@
+{% import 'utils.jinja' as utils %}
+
+CREATE TABLE PATIENT_LIFESPAN AS
+SELECT
+    concat('Patient/', id) AS ref,
+    {{ utils.date_with_fallback_suffix('birthDate', '-01-01T00:00:00') }} AS date_of_birth,
+    {{ utils.date_with_fallback_suffix('deceasedDateTime', '-12-31T23:59:59') }} + INTERVAL 30 DAY AS date_of_death
```

Comment:
Looks like Trino (and thus Athena) supports the INTERVAL keyword, but just chiming in here to say that you should run your metric in Athena for realz before landing, to confirm it works as expected (at minimum, doesn't blow up - if you're feeling fancy, actually find an example of it working as expected).

And - generally, the INTERVAL 30 days thing works fine when wrapping around years and months yeah? I assume your tests cover that (haven't looked yet), and it would be crazy for the syntax not to work in that case - but I always get nervous when I see any sort of date handling.